docs/Concepts/Privacy/Privacy-Overview.md

```diff
@@ -9,6 +9,9 @@ Other participants cannot access the transaction content or list of participants
 !!! important
     For production systems requiring private transactions, we recommend using a network 
     with a consensus mechanism supporting transaction finality. For example, [IBFT 2.0](../../HowTo/Configure/Consensus-Protocols/IBFT.md).
+    
+!!! Caution
+    Do not use [pruning](../Pruning.md) if you plan to make private transactions.
```

Comment:
I think we've talked about this before in a sync but to reduce the amount users have to read put the _if_ part of sentences first. Then if that part of the sentence doesn't apply to them, they don't need to read the dependant piece. eg, 

If make plan to use private transactions, do not use pruning. 

make -> use.  This constraint on pruning and private transactions applies when receiving private transactions as well.

Reply:
Updated